cluster/gce/gci/configure-helper.sh

```diff
@@ -2547,6 +2547,7 @@ function update-daemon-set-prometheus-to-sd-parameters {
 function update-event-exporter {
     local -r stackdriver_resource_model="${LOGGING_STACKDRIVER_RESOURCE_TYPES:-old}"
     sed -i -e "s@{{ exporter_sd_resource_model }}@${stackdriver_resource_model}@g" "$1"
+    sed -i -e "s@{{ exporter_sd_endpoint }}@${STACKDRIVER_TEST_ENDPOINT:-}@g" "$1"
```

Comment:
This variable is not used anywhere. There should be some default value specified.

Reply:
> This variable is not used anywhere.

I assume you're referring to `STACKDRIVER_TEST_ENDPOINT`? This is a kube-env var I added (CL just submitted). 

> There should be some default value specified.

The default value for `exporter_sd_endpoint` according to the above logic is just empty, which leaves the param `-endpoint` empty. This makes `event-exporter` [creates a `Service`](https://github.com/GoogleCloudPlatform/k8s-stackdriver/blob/master/event-exporter/sinks/stackdriver/sink_factory.go#L77) with the [default prod endpoint](https://github.com/googleapis/google-api-go-client/blob/master/logging/v2/logging-gen.go#L120). That's why I'm not specifically put the prod url in the default value.